tests/operators/test_base.py

```diff
@@ -0,0 +1,42 @@
+import pytest
+
+from cosmos.operators.base import (
+    DbtLSMixin,
+    DbtSeedMixin,
+    DbtRunOperationMixin,
+    DbtTestMixin,
+    DbtSnapshotMixin,
+    DbtRunMixin,
+)
+
+
+@pytest.mark.parametrize(
+    "dbt_command, dbt_operator_class",
+    [
+        ("test", DbtTestMixin),
+        ("snapshot", DbtSnapshotMixin),
+        ("ls", DbtLSMixin),
+        ("seed", DbtSeedMixin),
+        ("run", DbtRunMixin),
```

Comment:
The reason why `DbtRunOperationMixin` wasn't included here was because it's slightly different from the others since `base_cmd` is a property which includes the macro name:
https://github.com/astronomer/astronomer-cosmos/blob/3bc4d88d9815d8df941ff36e5070d692ed63a2fd/cosmos/operators/base.py#L331-L333

In the tests for this one I added a check for the `base_cmd` to make sure it's as expected: https://github.com/astronomer/astronomer-cosmos/blob/3bc4d88d9815d8df941ff36e5070d692ed63a2fd/tests/operators/test_base.py#L35-L42

I added a separate test for `DbtBaseOperator` like you suggested, thanks!